### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by andymcfee**
image

### CODE REVIEW

File: src/alto-ui/Form/CheckBox/CheckBox.scss [hunk 1]
```diff
@@ -1,33 +1,41 @@
 @import '../../scss/inc';
 
 .CheckBox {
-  margin-bottom: 10px;
+  & + & {
+    //margin-top: $spacing-small;
+  }
```
Comment (renaudtertrais) [inline src/alto-ui/Form/CheckBox/CheckBox.scss]: 🤔

File: src/alto-ui/Form/Radio/Radio.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+//   const { checked, disabled } = props;
+//   return (
+//     <RadioWrapper>
+//       <RadioInput {...props} type="radio" />
+//       <RadioLabel htmlFor={props.id} {...{ checked, disabled }}>
+//         {props.label}
+//       </RadioLabel>
+//     </RadioWrapper>
+//   );
+// };
```
Comment (renaudtertrais) [inline src/alto-ui/Form/Radio/Radio.js]: If you fix it, then you can remove this dead code

File: src/alto-ui/Form/CheckBox/CheckBox.scss [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
-  @extend %visually-hidden  
+  @extend %visually-hidden;
 }
 
 .CheckBox__label {
   font-size: $font-size-medium;
-  padding-left: 1.6em;
+  // font-size: 3rem;
+  line-height: 2em;
+  height: 2em;
```
Comment (renaudtertrais) [inline src/alto-ui/Form/CheckBox/CheckBox.scss]: we should use your new `$height-default` no ?

**Comment by renaudtertrais** [comment]
Also please rebase in order to fix your conflit first.

File: src/alto-ui/Form/CheckBox/CheckBox.scss (hunk 3 above)
Comment (andymcfee) [reply src/alto-ui/Form/CheckBox/CheckBox.scss]: No, cause this is `em` not `rem`

**Pull request merged**